docs/source/install/deb.rst

```diff
@@ -1,84 +1,210 @@
+* Take these steps on all boxes where you run stackstorm remote actions, **including** ``localhost``.
 
-   If you are feeling adventurous and want to attempt an installation using this method, we have left this documentation here for you to better understand the internals of the system as it currently exists
+  .. code-block:: bash
 
-   In the meantime, please feel free to take a look at many of our installation methods, including our `All-in-one Installer </install/all_in_one.rst>`_.
+    # Create an SSH system user
+    useradd stanley
+    mkdir -p /home/stanley/.ssh
+    chmod 0700 /home/stanley/.ssh
 
+    # Generate ssh keys on StackStorm box and copy over public key into remote box.
+    ssh-keygen -f /home/stanley/.ssh/stanley_rsa -P ""
+    cp ${KEY_LOCATION}/stanley_rsa.pub /home/stanley/.ssh/stanley_rsa.pub
 
-StackStorm APT Repositories
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
-This repository contains latest stable version of StackStorm components
-and dependencies:
+    # Authorize key-base acces
+    cat /home/stanley/.ssh/stanley_rsa.pub >> /home/stanley/.ssh/authorized_keys
+    chmod 0600 /home/stanley/.ssh/authorized_keys
+    chown -R stanley:stanley /home/stanley
 
-::
+    # Enable passwordless sudo
+    echo "stanley    ALL=(ALL)       NOPASSWD: SETENV: ALL" >> /etc/sudoers.d/st2
 
-  deb http://downloads.stackstorm.net/deb/ trusty_stable main
+* Enable passwordless sudo on for system user on StackStorm host
+  (required for local script actions, using ``local-shell-cmd`` and ``local-shell-script`` runners).
 
+  .. code-block:: bash
 
-This repository contains latest in development version of StackStorm components
-and dependencies:
+    echo "stanley    ALL=(ALL)       NOPASSWD: SETENV: ALL" >> /etc/sudoers.d/st2
 
-::
+* Adjust configuration in ``/etc/st2/st2.conf``:
 
-  deb http://downloads.stackstorm.net/deb/ trusty_unstable main
+  .. sourcecode:: ini
 
+    [system_user]
+    user = stanley
+    ssh_key_file = /home/stanley/.ssh/stanley_rsa
 
-Pre-requisites
-^^^^^^^^^^^^^^
+Check it all works
+~~~~~~~~~~~~~~~~~~
 
-APT dependencies
-''''''''''''''''
+  .. code-block:: bash
 
-::
+    st2 --version
 
-    aptlist='rabbitmq-server make python-virtualenv python-dev realpath python-pip mongodb mongodb-server gcc git mysql-server'
-    apt-get install -y ${aptlist}
+    st2 -h
 
-Pip dependencies
-''''''''''''''''
+    st2 action list --pack=core
 
-Use :github_st2:`requirements.txt </requirements.txt>` file from the release brunch of st2 source.
+    # List the actions from a 'core' pack
+    st2 action list --pack=core
```

Comment:
I don't think that is even invoked anywhere. https://github.com/StackStorm/st2/pull/2466 might do it. I am not sure.